homeassistant/components/fibaro/__init__.py

```diff
@@ -651,7 +651,11 @@ def extra_state_attributes(self):
                     self.fibaro_device.properties.batteryLevel
                 )
             if "armed" in self.fibaro_device.properties:
-                attr[ATTR_ARMED] = self.fibaro_device.properties.armed.lower() == "true"
+                armed = self.fibaro_device.properties.armed
+                if isinstance(armed, bool):
+                    attr[ATTR_ARMED] = armed
+                elif isinstance(armed, str) and armed.lower() in ("true", "false"):
+                    attr[ATTR_ARMED] = armed.lower() == "true"
```

Comment:
Should there be an else case here? Setting armed to none or false for unknown cases?

Reply:
I don't think so, if we cannot interpret the value it makes no sense to add it.